packages/top-app-bar/README.md

```diff
@@ -0,0 +1,88 @@
+# Top App Bar React
+
+MDC React Top App Bar acts as a React container around MDC Top App Bar, and implements MDC Top App Bar's foundation layer. Please see [MDC Top App Bar](https://github.com/material-components/material-components-web/tree/master/packages/mdc-top-app-bar).
```

Comment:
Try this:

MDC React Top App Bar is a React component which uses [MDC Top App Bar](https://github.com/material-components/material-components-web/tree/master/packages/mdc-top-app-bar)'s CSS and foundation JavaScript.